localisation/00_zuwc_united_localisation.csv

```diff
@@ -2994,13 +2994,32 @@ RULE_BLACK_EMPIRE_INVASION_ON;On;;An;;;;;;;;;;;x
+meteor_shard;Meteor Shard;;;;;;;;;;;;;x
+meteor_shard_desc;Dagger wielded by Arugal, former royal archmage of Gilneas and member of the Kirin Tor. Melee attacks have a chance to blast the target with Fire magic, making this weapon deceptively powerful.;;;;;;;;;;;;;x
+book_of_ur;The Book of Ur;;;;;;;;;;;;;x
+book_of_ur_desc;Written by Ur of Dalaran, this book contains a detailed account on the origins of the Worgen. "It is my hope that no Dalaran wizard seeks out the Worgen, even if done in light conscience. For no pact may be struck, no secrets may be learned, no good can come from these beasts. They are best left to their world. For if found in ours and not destroyed, our peril will be dire...";;;;;;;;;;;;;x
+EVTDESC_WCDRU_300.2;Difficult times have come to your lands. The threat of defeat in the war is significant. It could turn into an absolute disaster! It's not the first night you spend in your library thinking about it. Studying the origins of the Scythe of Elune, you stumble upon something that could save your country: knowledge of violent wolf-like creatures from the world of dreams, called worgen, and how to open a portal to their world. Their strength and ferocity can be used against your enemies. This is your chance!;;Schwierige Zeiten sind in Euer Land gekommen. Die Gefahr einer Niederlage im Krieg ist gro�. Es k�nnte sich in eine absolute Katastrophe verwandeln! Es ist nicht die erste Nacht, in der Ihr in Eurer Bibliothek verbringt und dar�ber nachdenkt. Untersuchung der Urspr�nge der Sense von Elune, das Euer Land retten k�nnte: das Wissen um brutale wolf�hnliche Kreaturen aus der Welt der Tr�ume, genannt Worgen, und wie man ein Portal zu ihrer Welt �ffnet. Ihre St�rke und Wildheit kann gegen Eure Feinde eingesetzt werden. Dies ist Eure Chance!;;;;;;;;;;;x
+EVTTITLE_WCDRU_304;Arugal's Journey;;;;;;;;;;;;;x
+EVTDESC_WCDRU_304;Dalaran has fallen. I must find a new benefactor, while there is still time to stop the Scourge from consuming every human north of the Thandol Span. I once served as royal archmage of Gilneas, and although those who reside north of the Greymane Wall are distrusted, the circumstances are dire enough that the royal court might be persuaded to act. If not, Pyrewood Village is only a short ride north, and Baron Silverlaine's nearby keep is ideally suited to serve as a staging area within Silverpine Forest. Alternatively, Hillsbrad is a short distance south of Dalaran, and I could help defend them if there are any survivors... Finally, I could offer my services to the king of Stromgarde. Thoradin's Wall may not be as formidable or well maintained as the Greymane Wall, but the Stromic warriors have a fearsome reputation and the mountains offer an easily defended refuge for civilians.;;;;;;;;;;;;;x
```

Comment:
>although those who reside north of the Greymane Wall

What if Genn didn't build a wall?

>Baron Silverlaine's nearby keep

What if different character rule in Pyrewood?